app/models/schedule.rb

```diff
@@ -3,17 +3,25 @@ class Schedule < ApplicationRecord
 
   attr_accessor :select
 
+  DAYS = [:monday, :tuesday, :wednesday, :thursday, :friday, :saturday, :sunday].freeze
+  TIMES = [:morning, :afternoon, :evening].freeze
+  CORRECT_SIZE = DAYS.size * TIMES.size
+
   def self.days
-    [:monday, :tuesday, :wednesday, :thursday, :friday, :saturday, :sunday]
+    DAYS
   end
 
   def self.times
-    [:morning, :afternoon, :evening]
+    TIMES
+  end
+
+  def self.correct_size
+    CORRECT_SIZE
```

Comment:
Can you remove these three methods and use the constants directly? I thought I wrote a comment about this but it seems to have disappeared.